torch/special/__init__.py

```diff
@@ -47,12 +48,12 @@
     'polygamma',
     'psi',
     'round',
+    'scaled_modified_bessel_k0',
```

Comment:
nit: there are a lot of shuffling of other names (in c++ code as well btw), reducing these would be nice to reduce noise.

Reply:
OK! I removed all of them.